Resources/views/menu.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'knp_menu.html.twig' %}
+
+{% block label %}
+    {%- set translation_domain = item.extra('translation_domain', 'messages') -%}
+    {%- set label = item.label -%}
+    {%- if translation_domain is not same as(false) -%}
```

Comment:
i think the option to set the translation domain to false to not translate at all is not documented.